docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -7,63 +7,74 @@ Developers from the community have built some excellent supplemental tools for w
 Do you have a project that you think should be featured here?
 Feel free to submit a pull request and the maintainers will work with you to write the description.
 
-## Premium tools
+## App starters
```

Comment:
I guess we can group those under this category.

Reply:
Alternative titles:

- Third-party scaffolds
- Third-party framework integrations

I think it's important to make it clear that we don't maintain anything on this page.

Reply:
Fair enough